lib/wrapper.js

```diff
@@ -135,8 +137,13 @@ var launch = function(){
 
   // Fork the child process
   var opts = {env:process.env};
+  var args;
+  if (argv.args) {
+    args = argv.args.split(' ');
```

Comment:
This call to `split` seems a bit dangerous. Should quoted arguments with space characters (such as `'1 2 "3 4"'`) be handled differently?